web/templates/recent_conversations_table.hbs

```diff
@@ -1,6 +1,6 @@
 <div id="recent_topics_filter_buttons" class="btn-group" role="group">
     <div id="recent_filters_group">
-        {{> recent_topics_filters}}
+        {{> recent_conversations_filters}}
```

Comment:
A good follow-up might be to move these all into a `web/templates/recent_converations/` directory; the top-level of `web/templates` is really cluttered so when we have a cluster a things that are part of a single larger component, it's usually good to do that. 

Best discussed briefly in #frontend and then done if there's consensus there.

Reply:
https://chat.zulip.org/#narrow/stream/6-frontend/topic/recent.20topics.20rename.20.2326651/near/1636738